plugins/inputs/ping/ping_test.go

```diff
@@ -41,6 +41,18 @@ PING www.google.com (216.58.218.164) 56(84) bytes of data.
 rtt min/avg/max/mdev = 35.225/43.628/51.806/5.325 ms
 `
 
+var enigma2PingOutput = `
```

Comment:
Add a comment here with the output of `ping -V` (or the equivalent version option).